cirq/testing/equals_tester.py

```diff
@@ -53,46 +62,19 @@ def verify_equality_group(self, *group_items: Any):
 
         # Within-group items must be equal.
         for v1, v2 in itertools.product(group_items, group_items):
-            # Binary operators should always work.
-            assert v1 == v2
-            assert not v1 != v2
-
-            # __eq__ and __neq__ should both be correct or not implemented.
-            assert hasattr(v1, '__eq__') == hasattr(v1, '__ne__')
-            # Careful: python2 int doesn't have __eq__ or __ne__.
-            if hasattr(v1, '__eq__'):
-                eq = NotImplemented
-                ne = NotImplemented
-                try:
-                    eq = v1.__eq__(v2)
-                    ne = v1.__ne__(v2)
-                except TypeError:
-                    pass
-                assert (eq, ne) in [(True, False),
-                                    (NotImplemented, False),
-                                    (NotImplemented, NotImplemented)]
-
-        # Check that this group's items don't overlap with other groups.
+            same = EqualsTester._eq_check(v1, v2)
```

Comment:
Make an attribute `self._equals_tester = EqualsTester()` and add the groups to it as they come in.

Reply:
Oops, was confused about where this was!